libs/pika/init_runtime/tests/unit/CMakeLists.txt

```diff
@@ -26,5 +26,5 @@ foreach(test ${tests})
     FOLDER ${folder_name}
   )
 
-  pika_add_unit_test("modules.init_runtime" ${test} ${${test}_PARAMETERS})
+  pika_add_unit_test("modules.init_runtime" ${test} VALGRIND)
```

Comment:
`*_PARAMETERS` is set above. The `pika_wait` test requires at least 2 worker threads. You can inline the `*_PARAMETERS` variable here if you'd like, but `THREADS <N >= 2>` cannot be removed completely.